recipes/pytest-localftpserver/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "pytest-localftpserver" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|replace("-", "_") }}-{{ version }}.tar.gz
+  sha256: 4a9ce95ff91aa87cc6325c7c2b31450bc6c42ee0e91e6ccba2223e36435f2c1c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - pyftpdlib >=1.2.0
+    - pyopenssl
+    - pytest
+
+test:
+  imports:
+    - pytest_localftpserver
```

Can we try to add ``pip check`` here please?
```suggestion
    - pytest_localftpserver
  requires:
    - pip
  commands:
    - pip check
```

---

@marcelotrevisani  :+1: sure thing! (Valeu pela revisão rápida!)

---

Tamo junto :smile: